integration-tests/kubernetes/src/it/knative/verify.groovy

```diff
@@ -0,0 +1,23 @@
+import io.dekorate.utils.Serialization
+import io.dekorate.deps.kubernetes.api.model.Container
+import io.dekorate.deps.kubernetes.api.model.KubernetesList
+import io.dekorate.deps.knative.serving.v1alpha1.Service;
+
+//Check that file exits
+String base = basedir
+File knativeYml = new File(base, "target/kubernetes/knative.yml")
+assert knativeYml.exists()
+
+//Check that its parse-able
+KubernetesList list = Serialization.unmarshal(knativeYml.text, KubernetesList.class)
+assert list != null
+
+// Un-comment once: https://github.com/fabric8io/kubernetes-client/issues/1838 is addressed.
```

Comment:
But this won't affect the correctness of the generated resources, correct? It's just that with the bug in the client you can't convert the yml into the domain object, right?

Reply:
:+1: